Fix table horizontal scrolling

aroralakshya opened the pull request:
Changed files:
	themes/original/css/common.css.php
	themes/pmahomme/css/common.css.php
Added style definition for .responsivetable element to prevent horizontal overflow
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

aroralakshya committed:
Fix for issue 13959
Changed files:
	themes/original/css/common.css.php
	themes/pmahomme/css/common.css.php
Added style definition for .responsivetable element to prevent horizontal overflow
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)

mauriciofauth commented on themes/original/css/common.css.php:
```diff
@@ -3323,6 +3323,10 @@
 }
 
+.responsivetable {
+    overflow-x: auto !important;
```
Using `!important` is generally not recommended. Could you solve it differently?

nulll-pointer replied:
It works without important, i just added it for safety reasons.

nulll-pointer replied:
i could remove it if you suggest.

aroralakshya committed:
Fix for issue #13959
Changed files:
themes/original/css/common.css.php
themes/pmahomme/css/common.css.php
Added style definition for .responsivetable element to prevent horizontal overflow
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)

mauriciofauth commented:
Merged, thanks for your contribution!
